src/docfx/cli/Docfx.cs

```diff
@@ -84,6 +85,16 @@ private static Command RestoreCommand()
             return command;
         }
 
+        private static Command ApplyTemplateCommand()
+        {
+            var command = CreateCommand("apply-template", "Apply templates", ApplyTemplate.Run);
+            DefineCommonCommands(command);
+            command.AddOption(new Option<string>("--locale", "Locale of the files."));
+            command.AddOption(new Option<string>("--input", "Input directory of structured json files."));
```

Comment:
This by convention is typically specified using `AddArgument`, which is `directory`. Remove this argument and use `CommandLineOptions.WorkingDirectory`.